app/graphql/users/type_defs.js

```diff
@@ -16,8 +18,9 @@ const rootTypes = gql`
 
 const customTypes = gql`
   type User {
-    firstName: String!
-    lastName: String!
+    name: String!
+    firstName: String @deprecated(reason: "Field is deprecated!")
```

Comment:
why is deprecated?

Reply:
That's the way it would be:
          "Field is deprecated and will be removed in the future Use name field instead"
??